Service Registry CLI guide (#308)
redhat-developer/app-services-guides: rhoas-cli-service-registry/README.adoc

> ////

Author: This is new service registry tutorial that bases on similar strategy that we have for kafka

:registry-product-long: OpenShift Service Registry
:registry: Service Registry
// Placeholder URL, when we get a HOST UI for the service we can put it here properly
:service-url: https://console.redhat.com/beta/application-services/streams/
:registry-url: https://console.redhat.com/beta/application-services/service-registry/
:property-file-name: app-services.properties
:rhoas-version: 0.29.0

// Other upstream project names
:samples-git-repo: https://github.com/redhat-developer/app-services-guides

//URL components for cross refs
:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
:getting-started-url: getting-started/README.adoc
:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
:kafkacat-url: kafkacat/README.adoc
:quarkus-url: quarkus/README.adoc
:rhoas-cli-url: rhoas-cli/README.adoc
:rhoas-cli-ref-url: commands
:topic-config-url: topic-configuration/README.adoc
:consumer-config-url: consumer-configuration/README.adoc
:service-binding-url: service-discovery/README.adoc

////
END GENERATED ATTRIBUTES
////

[id="chap-getting-started-rhoas-cli-registry"]

> = Getting started with the Service Registry using rhoas CLI

Reviewer: ```suggestion
= Getting started with the OpenShift Service Registry CLI (rhoas)
```

:context: getting-started-rhoas-registry

[IMPORTANT]
====
The `rhoas` command-line interface (CLI) is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
====

[role="_abstract"]
With the `rhoas` command-line interface (CLI), you can manage your application services from a terminal.
This guide describes how to get started quickly with Service Registry and the RHOAS CLI.

--
* Using `rhoas` CLI to manage Service Registry instances .

* Retrieve configuration to use Service Registry instance in your application

> * Manage Service Registry Artifacts directly from command line

Reviewer: ```suggestion
[role="_abstract"]
With the `rhoas` command-line interface (CLI), you can manage your application services from a terminal.

This guide describes how to get started quickly by using the `rhoas` CLI to create a Service Registry instance,  use it in an application, and manage its artifacts.

--

> [id="proc-installing-rhoas-registry{context}"]
> == Installing the rhoas CLI
> 
> [role="_abstract"]
> If you haven't installed RHOAS CLI you can install it using the following guide
>  TODO link to CLI installation guide
> 
> NOTE:  Service Registry integration requires *latest* version of RHOAS CLI
> 

Reviewer: ```suggestion
.Prerequisites
* You have downloaded and installed the latest version of the 'rhoas' CLI.
For instructions on installing the `rhoas` cli, see {base-url}{rhoas-cli-url}[_Installing the rhoas CLI _^]
```
